open-source-policy/practice.md

```diff
@@ -0,0 +1,242 @@
+# Practicing our open source policy #
```

Comment:
In `policy.md` titles are capitalized.  Here they are not.  I feel like we should be consistent.

I prefer the style used here.